### DISCUSSION TITLE
Table creation on the client

### DISCUSSION THREAD

**Discussion by maxrodro** [Mimetis/Dotmim.Sync#572] [Q&A]
Hi,

I have a question about cloning the structures, I have a new project and on the client side I have only 3 tables, but on the server side I have 10, when I synchronize the 10 server tables are created on the client, it is possible that not done?

**Comment by Mimetis** [comment]
I don't understand
How do u want to sync tables on client ,if they don't exists ?

**Reply by maxrodro** [reply]
Hi @Mimetis ,

Do not explain me well, on the server I have 10 tables, 3 of them are the ones that are synchronized to the client, the other 7 tables I do not want the structures to be created on the client, since they will never be synchronized.
I did it this way, but they are created on the client side.

syncSetup.Tables ["StockItems"]. SyncDirection = SyncDirection.None;

**Reply by Mimetis** [reply]
Well, you just have to define a SyncSetup with only these 3 tables to sync ?
